protostar-rust/src/main.rs

```diff
@@ -66,16 +77,38 @@ fn dependencies_for_package(
 fn main_execution() -> Result<()> {
     let _args = Args::parse();
 
+    let project_root_path = project_root::get_project_root()?
+        .to_str()
+        .expect("error reading project root path")
+        .to_owned();
     // TODO #1997
-    set_var("CARGO_MANIFEST_DIR", "../../cairo/Cargo.toml");
+    set_var(
+        "CARGO_MANIFEST_DIR",
+        format!("{}/../cairo/Cargo.toml", project_root_path),
```

Comment:
Why we need to add project root path here 🤔 
cc @cptartur

Reply:
We don't need to. This code was removed entirely on the master branch. This PR should be rebased.